cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/test/java/io/cdap/cdap/datapipeline/AutoJoinerTest.java

```diff
@@ -808,10 +813,13 @@ private void testSimpleAutoJoinUsingSQLEngineWithCapabilities(List<String> requi
     MockSource.writeInput(inputManager, purchaseData);
 
     WorkflowManager workflowManager = appManager.getWorkflowManager(SmartWorkflow.NAME);
-    Map<String, String> args = Collections.singletonMap(MockAutoJoiner.PARTITIONS_ARGUMENT, "1");
+    Map<String, String> args = ImmutableMap.<String, String>builder()
+      .put(MockAutoJoiner.PARTITIONS_ARGUMENT, "1")
+      .put(io.cdap.cdap.etl.common.Constants.CONSOLIDATE_STAGES, "false")
```

Comment:
Please use an import for this

Reply:
`Constants` is already imported in this file, thus I have to use the fully qualified package name.

Reply:
Got you, sounds good